From fad4837c958d8d1ec751372dd3b64984aa207f0a Mon Sep 17 00:00:00 2001
From: JotaRandom <prflr88@gmail.com>
Date: Thu, 18 Sep 2025 19:48:18 -0300
Subject: [PATCH] Update build.xml

Fix for Build Issues

Update build.xml

Modern versions of Java changed the way this is handled, and also
changed the "minimal" version acceptable, this is a temporary fix
to allow micropolis-java to build agains modern java versions.
Is still a "Deprecated" way to fix it but at leas now it builds.
---
 build.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.xml b/build.xml
index 6cf025f..b320d93 100644
--- a/build.xml
+++ b/build.xml
@@ -83,7 +83,7 @@
        classpathref="build-classpath"
 	includeantruntime="false"
 	debug="true" debuglevel="lines,vars,source"
-	source="1.6" target="1.6"
+	release="8" source="8" target="8"
        >
 	<compilerarg value="-Xlint:unchecked" />
 	<compilerarg value="-Xlint:deprecation" />
-- 
2.51.0

